### ISSUE TITLE
Add CHANGELOG.md file to generated projects

### ISSUE THREAD

**Issue by MrSlide** [imperodesign/generator-impero#40]
Change logs are useful, especially on long running projects that go through many developers.

http://keepachangelog.com

**Comment by SamHH** [comment]
For me, the projects we build seldom ever go through "versions", and you can tell most things apart by commit history. Again though, I'm open to it. Thoughts everyone else?

P.S. Sorry if I've seemed negative with most of my replies to today's issues. I've thought about many of these things already and formed an opinion about them. But they can be changed or even overruled! haha :-)

**Comment by MrSlide** [comment]
Commit history is not a good way to tell what has been added/removed/changed and when. Sometimes there are many commits just for one feature, and some commit messages can be a bit cryptic. Also, the changes don't necessarily get released when they are created.

I also didn't do release versions or change logs until this one big project that I worked on, but since then I gained a better appreciation for the value of change logs and release versions.

It's particularly useful when there are breaking changes in the code, or when something stops working but people only notice it a few releases later.

I'm not saying that all projects must have a change log,  but instead I'm suggesting that we include the option to include a change log.
